infra/src/main/java/com/redhat/developer/infra/dto/BridgeStatus.java

```diff
@@ -3,5 +3,7 @@
 public enum BridgeStatus {
     REQUESTED,
     PROVISIONING,
-    AVAILABLE
+    AVAILABLE,
+    DELETION_REQUESTED,
```

Comment:
I choosed `DELETION_REQUESTED` so to stay in line with `REQUESTED`. Because the manager is not really `DELETING` the instance yet, and the `REQUESTED` part gives the idea that the shard will have to fetch those instances to take actions. 

If it's fine for you, I'll keep it as it is and refactor it in the near future

Reply:
https://issues.redhat.com/browse/MGDOBR-28